geode-for-redis/src/main/java/org/apache/geode/redis/internal/data/RedisSet.java

```diff
@@ -74,12 +74,26 @@ public RedisSet(int expectedSize) {
    */
   public RedisSet() {}
 
+  public static Set<byte[]> sunion(RegionProvider regionProvider, List<RedisKey> keys) {
+    MemberSet result = calculateUnion(regionProvider, keys, true);
+    return getSetOpResult(result);
```

Comment:
I think it should be fine to just use 
```
return calculateUnion(regionProvider, keys, true);
```
here, since that never returns null.

Ideally, none of the calculate* methods should return null, but rather an empty MemberSet which can then safely be returned from the calling method, removing the need for the `getSetOpResult()` method.